registry.json

```diff
@@ -3871,5 +3871,17 @@
     "telegram": "w_o_u_t",
     "operatorName": "Wout",
     "github": "tiligallery"
+  },
+  {
+    "ticker": "TDOT",
+    "poolId": "8833a263b80491140592cb907f46add5e1a1ddbce66725ce7675e2af",
+    "social": {
+      "website": "https://ada.tdot.finance/",
+      "twitter": "TDOTStaking",
+      "facebook": "",
+      "instagram": "tdot_staking",
+      "youtube": ""
+    },
+    "telegram": "tdotstaking"
```

Comment:
Please can you change this to the telegram handle of an individual and not a channel or group? Also please can you add in `operatorName` and `github` keys, even if the values are blank? Thanks